src/apps/bm_devkit/hello_world/CMakeLists.txt

```diff
@@ -268,6 +263,7 @@ set(BM_CORE_LWIP_INCLUDES
     ${SRC_DIR}/lib/lwip
     ${LWIP_INCLUDE_DIRS}
     ${BCMP_INCLUDES}
+    ${BRISTLEMOUTH_INCLUDES}
```

Comment:
This is needed right now because of bm_core's reliance on `bm_common_messages` repository.
I think we should discuss how to handle this in our Monday meeting.
It should be a submodule of `bm_core` but it is full of cpp files.